sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -33,53 +35,54 @@ import life.qbic.portal.sampletracking.components.projectoverview.subscribe.Subs
  *
 */
 @Log4j2
-class ProjectOverviewView extends VerticalLayout {
+class ProjectOverviewView extends VerticalLayout implements HasHotbar, HasTitle {
+
+    private static final String TITLE = "Project Overview"
 
     private final ProjectOverviewViewModel viewModel
     private final DownloadProjectController downloadProjectController
     private final SubscribeProjectController subscribeProjectController
     private final NotificationService notificationService
     private final FailedQCSamplesView failedQCSamplesView
-    private final ProjectOverviewController projectOverviewController
+    private final FailedQCSamplesController failedQCSamplesController
 
     private Grid<ProjectSummary> projectGrid
     private HorizontalSplitPanel splitPanel
     private static final Collection<String> columnIdsWithFilters = ["ProjectCode", "ProjectTitle"]
 
     final static int MAX_CODE_COLUMN_WIDTH = 400
-    final static int MAX_STATUS_COLUMN_WIDTH = 200
     private FileDownloader fileDownloader
+    private HorizontalLayout projectButtonBar = new HorizontalLayout()
 
     ProjectOverviewView(NotificationService notificationService, ProjectOverviewViewModel viewModel, DownloadProjectController downloadProjectController
-                        , FailedQCSamplesView failedQCSamplesView, ProjectOverviewController projectOverviewController, SubscribeProjectController subscribeProjectController){
+                        , FailedQCSamplesView failedQCSamplesView, FailedQCSamplesController failedQCSamplesController, SubscribeProjectController subscribeProjectController){
         this.notificationService = notificationService
         this.viewModel = viewModel
         this.downloadProjectController = downloadProjectController
         this.subscribeProjectController = subscribeProjectController
         this.failedQCSamplesView = failedQCSamplesView
-        this.projectOverviewController = projectOverviewController
+        this.failedQCSamplesController = failedQCSamplesController
 
         initLayout()
     }
 
     private void initLayout(){
-        Label titleLabel = new Label("Project Overview")
-        titleLabel.addStyleName(ValoTheme.LABEL_HUGE)
 
         Label spacerLabel = new Label("&nbsp;", ContentMode.HTML)
 
         setupProjects()
 
-        HorizontalLayout projectButtonBar = setupButtonLayout()
-        VerticalLayout projectLayout = new VerticalLayout(projectButtonBar,projectGrid)
+        setupButtonLayout(projectButtonBar)
+        VerticalLayout projectLayout = new VerticalLayout(projectGrid)
         projectLayout.setMargin(false)
 
         splitPanel = createSplitLayout(projectLayout,failedQCSamplesView)
         failedQCSamplesView.addVisibilityChangeListener({ splitPanel.splitPosition = it.newValue ? 65 : 100 })
 
         connectFailedQcSamplesView()
         bindManifestToProjectSelection()
-        this.addComponents(titleLabel, spacerLabel, splitPanel)
+        this.addComponents(spacerLabel, splitPanel)
```

Comment:
Since you changed the layout this spacerlabel should be moved. It is responsible to add space between title and hotbar :)